src/components/FeedPost/FeedPost.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../styles/Mixins';
+@import '../../styles/Colors';
+@import '../../styles/FontSizes';
+@import '../../styles/FontWeights';
+@import '../../styles/Breakpoints';
+
+.feedPost__container{
```

Comment:
look at other files and components. There is always one root clas like "feedPost" and below there is somethig like this

```
.feedPost {
&__container {
...
}
&__user {
..
}
}
```

It should be definitely refactored